src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V68/Eth68ProtocolHandler.cs

```diff
@@ -0,0 +1,149 @@
+using System.Diagnostics;
+using System.Linq;
+using Nethermind.Consensus;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Specs;
+using Nethermind.Logging;
+using Nethermind.Network.P2P.Subprotocols.Eth.V65;
+using Nethermind.Network.P2P.Subprotocols.Eth.V67;
+using Nethermind.Network.P2P.Subprotocols.Eth.V68.Messages;
+using Nethermind.Network.Rlpx;
+using Nethermind.Serialization.Rlp;
+using Nethermind.Stats;
+using Nethermind.Synchronization;
+using Nethermind.TxPool;
+
+namespace Nethermind.Network.P2P.Subprotocols.Eth.V68;
+
+public class Eth68ProtocolHandler : Eth67ProtocolHandler
+{
+    private readonly IPooledTxsRequestor _pooledTxsRequestor;
+
+    public override string Name => "eth68";
+
+    public override byte ProtocolVersion => 68;
+
+    public Eth68ProtocolHandler(ISession session,
+        IMessageSerializationService serializer,
+        INodeStatsManager nodeStatsManager,
+        ISyncServer syncServer,
+        ITxPool txPool,
+        IPooledTxsRequestor pooledTxsRequestor,
+        IGossipPolicy gossipPolicy,
+        ISpecProvider specProvider,
+        ILogManager logManager)
+        : base(session, serializer, nodeStatsManager, syncServer, txPool, pooledTxsRequestor, gossipPolicy,
+            specProvider, logManager)
+    {
+        _pooledTxsRequestor = pooledTxsRequestor;
+    }
+
+    public override void HandleMessage(ZeroPacket message)
+    {
+        switch (message.PacketType)
+        {
+            case Eth68MessageCode.NewPooledTransactionHashes:
+                NewPooledTransactionHashesMessage68 newPooledTxHashesMsg =
+                    Deserialize<NewPooledTransactionHashesMessage68>(message.Content);
+                ReportIn(newPooledTxHashesMsg);
+                Handle(newPooledTxHashesMsg);
+                break;
+            default:
+                base.HandleMessage(message);
+                break;
+        }
+    }
+
+    private void Handle(NewPooledTransactionHashesMessage68 message)
+    {
+        if (message.Hashes.Count != message.Types.Count || message.Hashes.Count != message.Sizes.Count)
+        {
+            if (Logger.IsTrace)
+                Logger.Trace($"Wrong format of {nameof(NewPooledTransactionHashesMessage68)} message. " +
+                             $"Hashes count: {message.Hashes.Count} " +
+                             $"Types count: {message.Types.Count} " +
+                             $"Sizes count: {message.Sizes.Count}");
+
+            throw new SubprotocolException($"Wrong format of {nameof(NewPooledTransactionHashesMessage68)} message. " +
+                                           $"Hashes count: {message.Hashes.Count} " +
+                                           $"Types count: {message.Types.Count} " +
+                                           $"Sizes count: {message.Sizes.Count}");
+        }
+
+        Metrics.Eth68NewPooledTransactionHashesReceived++;
+
+        Stopwatch stopwatch = Stopwatch.StartNew();
```

Comment:
Is this something we want to benchmark in prod code here?

Reply:
This is code from NewPooledTransactions from eth65, maybe it's not needed anymore